health/dl-error-spikes.js

```diff
@@ -0,0 +1,21 @@
+const nHealth = require('n-health');
+const samplePeriod = 10; // minutes
+const threshold = 5;
+
+// 3 and 5 are the positions of wildcards in the chain, 0 based
+// 6 is the node the count is based on, 0 based
+const metric = 'aliasByNode(sumSeriesWithWildcards(next.heroku.syndication-dl.*.express.* _{ GET, POST, PUT }.res.status_5xx.count, 3, 5), 6)';
```

Comment:
```suggestion
const metric = 'aliasByNode(sumSeriesWithWildcards(next.heroku.syndication-dl.*.express.* _{ GET, POST, PUT }.res.status_4xx.count, 3, 5), 6)';
```

Shouldn't it be `4xx`

Reply:
Yes, I was looking too hard at the example Umberto posted. D'oh.